learning_assistant/views.py

```diff
@@ -39,6 +43,19 @@
 class CourseChatView(APIView):
     """
     View to retrieve chat response.
+
+    Accepts: [POST]
+
+    Path: /learning_assistant/v1/course_id/{course_run_id}
+
+    Parameters:
+        * course_run_id: the ID of the course
+
+    Responses:
+        * 200: OK
+        * 400: Malformed Request - Course ID is not a valid course ID.
+        * 403: Forbidden - Learning assistant not enabled for course or learner does not have a valid enrollment or is
+                           not staff.
```

Comment:
I'm not sure if you're seeing something different than what I'm seeing, but I tend to align subsequent lines of paragraphs to the beginning character of the paragraph, because I find it easier to read. Would you prefer a different indentation?